docs/en_US/Compression/pruning.rst

```diff
@@ -0,0 +1,26 @@
+#################
+Pruning
+#################
+
+A common technique to increase sparsity in neural network model weights and activations is pruning.
+The pruning methods explore the redundancy in the model weights(parameters) and try to remove/prune the redundant and uncritical weights.
+The redundant elements are pruned from the model, their values are zeroed and we make sure they don't take part in the back-propagation process.
```

Comment:
It's better to use the same terminology (elements -> weights)

Reply:
Thanks, fixed.